nixos/tests/forked-daapd.nix

```diff
@@ -0,0 +1,44 @@
+import ./make-test-python.nix ({ pkgs, ... }:
```

Comment:
I'm not sure whether this is something that needs fixing:

https://github.com/NixOS/nixpkgs/pull/85550/checks?check_run_id=1432921834

@tfc Any idea?

Reply:
log says

```
gcc: error: unrecognized command line option '-m64'
make[1]: *** [Makefile:453: antlr3baserecognizer.lo] Error 1
make[1]: Leaving directory '/build/libantlr3c-3.4'
make: *** [Makefile:308: all] Error 2
error: --- Error --- nix-daemon
builder for '/nix/store/g93kbabwq81cx2r7hcpbvc7cy69qc0h4-libantlr3c-3.4.drv' failed with exit code 2; last 10 log lines:
  building
  build flags: SHELL=/nix/store/93xwsgby6fchr8a58ckgcm2lf4nkv6fl-bash-4.4-p23/bin/bash
  make  all-am
  make[1]: Entering directory '/build/libantlr3c-3.4'
  /nix/store/93xwsgby6fchr8a58ckgcm2lf4nkv6fl-bash-4.4-p23/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -Iinclude    -m64  -O2  -Wall -c -o antlr3baserecognizer.lo `test -f 'src/antlr3baserecognizer.c' || echo './'`src/antlr3baserecognizer.c
  libtool: compile:  gcc -DHAVE_CONFIG_H -I. -Iinclude -m64 -O2 -Wall -c src/antlr3baserecognizer.c  -fPIC -DPIC -o .libs/antlr3baserecognizer.o
  gcc: error: unrecognized command line option '-m64'
  make[1]: *** [Makefile:453: antlr3baserecognizer.lo] Error 1
  make[1]: Leaving directory '/build/libantlr3c-3.4'
  make: *** [Makefile:308: all] Error 2
```

Seems like it's not the test but `libantlr`'s build process